app/services/keybase_service.rb

```diff
@@ -0,0 +1,27 @@
+class KeyBaseService
+
+  include AgentLogic
+
+  def initialize
+    @keybase_url = "https://keybase.io/"
+  end
+
+  def get_validator_avatar(validator)
```

Comment:
I'd recommend changing this method name. When I see a method that has `get` in it, I'd think it's **only** retrieving something. Whereas your method gets _and_ sets `Validator#url`